x-pack/plugins/data_visualizer/public/application/common/components/field_types_filter/field_types_help_popover.tsx

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiBasicTable,
+  EuiFilterButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiLink,
+  EuiPanel,
+  EuiPopover,
+  EuiPopoverTitle,
+  EuiText,
+  useEuiTheme,
+} from '@elastic/eui';
+import React, { FC, useMemo, useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FieldIcon } from '@kbn/react-field';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { getFieldTypeDescription } from '../../../../../common/constants';
+import { useDataVisualizerKibana } from '../../../kibana_context';
+
+interface FieldTypeTableItem {
+  id: number;
+  dataType: string;
+  description: string;
+}
+
+export const FieldTypesHelpPopover: FC<{
+  fieldTypes: string[];
+}> = ({ fieldTypes }) => {
+  {
+    const { services } = useDataVisualizerKibana();
+    const { docLinks } = services;
+    const { euiTheme } = useEuiTheme();
+
+    const [isHelpOpen, setIsHelpOpen] = useState(false);
+
+    const onHelpClick = () => setIsHelpOpen((prevIsHelpOpen) => !prevIsHelpOpen);
+    const closeHelp = () => setIsHelpOpen(false);
+
+    const items: FieldTypeTableItem[] = useMemo(() => {
+      return fieldTypes.map((type, index) => ({
+        id: index,
+        dataType: type,
+        description: getFieldTypeDescription(type, docLinks),
+      }));
+    }, [fieldTypes, docLinks]);
+
+    const columnsSidebar = [
+      {
+        field: 'dataType',
+        name: i18n.translate('xpack.dataVisualizer.fieldTypesPopover.dataTypeColumnTitle', {
+          defaultMessage: 'Data type',
+        }),
+        width: '110px',
+        render: (name: string) => (
+          <EuiFlexGroup alignItems="center" responsive={false} gutterSize="xs">
+            <EuiFlexItem grow={false}>
+              <FieldIcon type={name} />
+            </EuiFlexItem>
+            <EuiFlexItem>{name}</EuiFlexItem>
+          </EuiFlexGroup>
+        ),
+      },
+      {
+        field: 'description',
+        name: i18n.translate('xpack.dataVisualizer.fieldTypesPopover.descriptionColumnTitle', {
+          defaultMessage: 'Description',
+        }),
+        // eslint-disable-next-line react/no-danger
+        render: (description: string) => <div dangerouslySetInnerHTML={{ __html: description }} />,
```

Comment:
The description here is only ever set internally, right? Just making sure we want use the set inner html

Reply:
I realized we don't even need it so I removed that part here https://github.com/elastic/kibana/pull/137121/commits/024eac1c32383dd62b33217f38329083cbb2ecd6